### ISSUE TITLE
Jump-to-line from terminal regression in supported formats.

### ISSUE THREAD

**Issue by rossc719g** [microsoft/vscode#190533]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.81.0 (Universal). Commit: 6445d93c81ebe42c4cbd7a60712e0b17d9463e97
- OS Version: MacOS 13.4.1 (c) (22F770820d)

In previous versions of vscode (within the last month or so?) Lines in the terminal like this:
```Error: "path/to/a/file.foo", line 49, column 16: Some error message```
used to generate links to specific lines & columns in a file. Specifically, the `"path/to/a/file.foo", line 49, column 16` portion used to be a single link to that location.  But with the latest version, it only identifies `path/to/a/file.foo` as a link and does not jump to the line or column indicated.

Can this behavior be restored?

Steps to Reproduce:
1. Open a file
2. In the terminal create that message, e.g.:  `echo 'Error: "path/to/a/file.foo", line 49, column 16: Some error message'`

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.81.1. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by rossc719g** [comment]
Yep, the same behavior still occurs in Version 1.81.1 (Universal). Commit: 6c3e3dba23e8fadc360aed75ce363ba185c49794

**Comment by rossc719g** [comment]
Very interesting.

I was seeing the bad behavior with my normal setup, and it had persisted for quite some time, including several reloads of the window. (E.g., clicking the `reload required` button on an extension).

Yesterday when I was filing this, I restarted without extensions, (since the issue form told me to), and still saw the bad behavior.

Then based on the comment from VSCodeTriageBot, I upgraded from 1.82.0 to 1.82.1, and still saw the same bad behavior.

Just now, I tried again and saw the bad behavior.

Based on your comment, I checked and found that I was not in the right dir. I.e., if the path was `foo/bar/baz.txt`, the cwd of the terminal was the `foo` dir.   So, I went up a dir (`cd ..`), and tried again, and still got the bad behavior.

I closed the terminal and created a new one, and then it worked correctly.

Because I was curious I moved into a subdir (`cd foo` in my example), and tried again, and again got the correct behavior.

I even did another "Reload Window", and it still did the correct behavior, even though I was still in the subdir.

So.. all I can conclude is that there was something (other than the cwd, apparently?) about my terminal, which persisted through multiple restarts, and an upgrade, which was causing the issue....

I guess we can close this issue now, but I would love to know what is going on here.  The behavior is apparently more subtle than I would have guessed.
